globus_sdk/base.py

```diff
@@ -49,7 +49,7 @@ class BaseClient:
         string, and may be useful when debugging issues with the Globus Team
     :type app_name: str
     :param http_timeout: Number of seconds to wait on HTTP connections. Default is 60.
-        A value of -1 indicates that no timeout should be used (requests can hang
+        A value of `None` indicates that no timeout should be used (requests can hang
```

Comment:
This needs fixing, though it's non-obvious. `None` is the default value, and gets ignored by env var parsing, so `None` --> a timeout of 60s. I want to support an explicit `None` to mean this. I'll see what I can do in a follow-up right now.

Reply:
Okay, this change has been reverted, so ``-1`` still means no-timeout.